app/env.ts

```diff
@@ -1,7 +1,7 @@
 export function getEnv() {
     return {
-        PROFILE_URL: process.env.PROFILE_URL || "https://lens.k3l.io",
-        CONTENT_URL: process.env.CONTENT_URL || "https://content.lens.k3l.io",
+        PROFILE_URL: "https://lens.k3l.io",
+        CONTENT_URL: "https://content.lens.k3l.io",
     }
```

Comment:
I feel like the problem is in Vercel and may not be in the code. According to this doc - [Declare an Environment Variable](https://vercel.com/docs/projects/environment-variables#declare-an-environment-variable) using process.env seems to be the right way to go. Can you please confirm this ?

Reply:
I believe the problem is that we are trying to load the environment variables on the client component not server rendered component. The client side component that has `useEffect` can only be referenced from client, and we are calling this call from the client.

I've checked the environment variables, and we already have them imported on Vercel. 

I'm not too familiar with Remix, and let me investigate how they work on this.

Reply:
https://remix.run/docs/en/main/guides/envvars#browser-environment-variables

Reply:
yes, that's why i was thinking a better fix would be to move the call to getEnv into the `export default function HeaderLinks()` function in HeaderLinks.tsx and into `export default function Index()` in the index.tsx

Reply:
I think `HeaderLinks` and `Index` are still a client component, and the proper way to solve this problem is to shove it into window like this link. https://remix.run/docs/en/main/guides/envvars#browser-environment-variables